apps/io-wallet-user-func/src/app/config.ts

```diff
@@ -50,6 +51,50 @@ export const ANDROID_CRL_URL =
 export const ANDROID_PLAY_INTEGRITY_URL =
   "https://www.googleapis.com/auth/playintegrity";
 
+export const MailConfig = t.type({
+  mailFeatureFlag: t.boolean,
+  mailSender: t.intersection([t.string, NonEmptyString]),
+  mailupSecret: t.intersection([t.string, NonEmptyString]),
+  mailupUsername: t.intersection([t.string, NonEmptyString]),
+});
```

Comment:
Why do you use the intersection of both `t.string` and `NonEmptyString`?

Reply:
So that I don't use any casting (keyword `as`), or some "convenience solution" when passing the configuration to `getMailerTransporter()`. Effectively, `string | INonemptyStringTag` (the last one is given by NonEmptyString) is the type expected by some properties of the `getMailerTransporter()` configuration.

Reply:
I checked and MailConfig only requires NonEmptyString https://github.com/pagopa/io-functions-commons/blob/1cf04d9ce5a96e7f7bbe8667cf8c818530c0b28c/src/mailer/config.ts#L30 You should structure this part better and eliminate the intersection